src/main/java/wolox/gmjavatraining/models/Book.java

```diff
@@ -0,0 +1,54 @@
+package wolox.gmjavatraining.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Getter @Setter
```

Comment:
Check the @Data annotation and adding, if necessary, use an annotation to ignore the automated @getter @setter for the id column, don't recall which one it was. Let me know if you can't find it on google.